### ISSUE TITLE
Is "mlContext.Transforms.Conversion.MapKeyToValue" call supported in 0.8 version?

### ISSUE THREAD

**Issue by sakopov12** [dotnet/machinelearning#1981]
Hello,

I have installed the 0.8 version, and can see that mlContext.Transforms.Conversion.MapKeyToValue() gives me compiler error. What would be the alternative call?

Thank you.

**Comment by yaeldekel** [comment]
Hi @sakopov12 , thanks for your question! MLContext.Transforms.Conversion.MapKeyToValue should be available in version 0.8. Could you provide more details about your problem, such as a larger code snippet, or the error message from the compiler error that you are hitting?

**Comment by Zruty0** [comment]
I just looked it up in the source: `mlContext.Transforms.Conversion.MapKeyToValue()` is legal in the current daily. I am not sure about 0.8, but you can always call `new KeyToValueMappingEstimator(mlContext, ...)` as an alternative.

**Comment by sakopov12** [comment]
I have double checked if the code I posted was exactly the same as what I saw when working at home, and I see that I was incorrect. The code that gives me an error looks like this:
mlContext.Transforms.Categorical.MapKeyToValue().
I was going through an article in MSDN magazine: https://msdn.microsoft.com/magazine/mt848634, and the sample had this code. The other type that is giving me an error is KeyToValueEstimator.

Sorry for the confusion, and thank you for getting back to me!
